optaplanner-examples/src/main/java/org/optaplanner/examples/cloudbalancing/domain/CloudBalance.java

```diff
@@ -54,6 +53,8 @@ public void setProcessList(List<CloudProcess> processList) {
     }
 
     @PlanningScore
+    @JsonDeserialize(using = HardSoftScoreJacksonDeserializer.class)
+    @JsonSerialize(using = HardSoftScoreJacksonSerializer.class)
```

Comment:
We could possibly avoid the (de)serializers if we register the `OptaPlannerJacksonModule` in the `JacksonSolutionFileIO`; something we should probably do anyway.